rails/attributes/customize.rb

```diff
@@ -1 +1,3 @@
-normal[:rails][:version] = "4.1.7"
+normal[:rails][:version] = '4.1.7'
+
+node[:deploy].delete_if { |application, _| !node[:rails].includes?(application) }
```

Comment:
OpsWorks deploys every application in the `node[:deploy]` node which isn't what we want with our separate applications.   In order to fix this, temporarily, is to remove items from the node we don't want deployed.